azkaban-web-server/src/web/js/azkaban/view/flow-execute-dialog.js

```diff
@@ -56,15 +56,24 @@ azkaban.FlowExecuteDialogView = Backbone.View.extend({
     var successEmailsOverride = $("#override-success-emails").is(':checked');
 
     var flowOverride = {};
+    var jobOverride = {};
     var editRows = $(".editRow");
     for (var i = 0; i < editRows.length; ++i) {
       var row = editRows[i];
+      var jobOrFlow = row.cells[0].firstChild.value;
       var td = $(row).find('span');
       var key = $(td[0]).text();
       var val = $(td[1]).text();
 
       if (key && key.length > 0) {
-        flowOverride[key] = val;
+        if (jobOrFlow && jobOrFlow.length > 0) {
+          if (!jobOverride.hasOwnProperty(jobOrFlow)) {
+            jobOverride[jobOrFlow] = {};
+          }
```

Comment:
This `if` can be replaced with: 
jobOverride[jobOrFlow] = jobOverride[jobOrFlow] || {}

Reply:
>This `if` is equivalent to:
> `jobOverride[jobOrFlow] = jobOverride[jobOrFlow] || {}`

Nice, did that.